crates/breez-sdk/core/src/persist/sqlite.rs

```diff
@@ -187,6 +187,12 @@ impl SqliteStorage {
              FROM payments;",
             "DROP TABLE payments;",
             "ALTER TABLE payments_new RENAME TO payments;",
+            "CREATE TABLE payment_details_spark (
+              payment_id TEXT PRIMARY KEY,
+              invoice_details TEXT,
```

Comment:
Do we need to store the whole invoice details object? vs the essentials (description and invoice string)

Reply:
Not really. In any case all data can be recovered from the invoice string. Changed to store just description and invoice string as suggested